### PR TITLE
Support for Traditional Chinese Pokémon names

### PR DESCRIPTION
**Pull request by CalvertYang**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ Pokémon names are available for the following languages:
 - `ja` (Japanese)
 - `zh-Hans` (Simplified Chinese)
 - `ru` (Russian)
+- `zh-Hant` (Traditional Chinese)
```
Comment (SamVerschueren) [inline readme.md]: Put the simplified chinese line right above this one to "group" them together.

**Comment by SamVerschueren** [comment]
Use `tabs` instead of `spaces` in the pokemon json file.

File: index.js [hunk 2]
```diff
@@ -14,7 +14,8 @@ const languages = new Set([
 	'fr',
 	'ja',
 	'zh-Hans',
-	'ru'
+	'ru',
+	'zh-Hant'
```
Comment (SamVerschueren) [inline index.js]: Move `zh-Hans` down here as well.

File: test.js [hunk 3]
```diff
@@ -70,6 +70,12 @@ test('Get Russian name by ID', testNames, 'ru', [
 	'Вулканион'
 ]);
 
+test('Get Traditional Chinese name by ID', testNames, 'zh-Hant', [
```
Comment (SamVerschueren) [inline test.js]: Move the test up under the simplified chinese test.

**Comment by SamVerschueren** [comment]
Sorry for the comments, should be good for merge after these ones get resolved. I'll let @sindresorhus have a look as well.

**Comment by CalvertYang** [comment]
@SamVerschueren Thanks for your reviewing.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**